<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629695941723' stop='1629696165719'>
    <name>Create new project </name>
    <title>Create new project </title>
    <test-cases>
        <test-case start='1629695961833' status='failed' stop='1629696165428'>
            <name>Should create new project successfully.</name>
            <title>Should create new project successfully.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='bf021d3c-d693-4844-8de9-f7f9c595d64b-attachment.png' type='image/png' size='59097'/>
                <attachment title='Screenshot' source='7adb8c7c-e47a-49e1-8b73-e6b17cfa74f0-attachment.png' type='image/png' size='59097'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>